graphs/directed_and_undirected_(weighted)_graph.py

```diff
@@ -226,7 +226,9 @@ def has_cycle(self):
                                 break
                             else:
                                 return True
-                                anticipating_nodes.add(stack[len_stack_minus_one])
+                                # TODO:The following code is unreachable.
+                                anticipating_nodes.add(
+                                    stack[len_stack_minus_one])
```

Comment:
psf/black will undo this change.

Reply:
yeah autopep8 did that, i've just pushed after running black